app/Http/Controllers/Controller.php

```diff
@@ -246,6 +246,25 @@ public function pdf()
             return back()->with('error', 'Could not parse JSON data. Response was ' . substr(trim($response->body()), 0, 100) . '…')->withInput();
         }
 
+        // Where $meetings[n]['day'] is an array, create separate values for each day
+        foreach ($meetings as $key => $entry) {
+            if (is_array($entry['day'])) {
+                // Remove non zero empty values (NULL, FALSE & '')
+                $days = array_filter($entry['day'], 'strlen');
+                $first = reset($days);
```

Comment:
is this the same as saying `$first = $days[0];`?

Reply:
It would net you the same result, yes. I used reset instead of $days[0] in the edge case the array isn't numerically indexed for some reason.